pkg/opensearch/snapshot_tests/testdata/lucene_logs.query_input.json

```diff
@@ -7,14 +7,13 @@
     "alias": "",
     "bucketAggs": [
       {
-        "field": "timestamp",
```

Comment:
is there a reason that the field changed?

Reply:
yes, sorry this should have also gone into an explanatory comment. this also shows that no matter the field put here, the "defaultTimeField" will be used over here https://github.com/grafana/opensearch-datasource/pull/277/files/98a8fa01a4bf2aa0adc11573355c0143f38a1c99#diff-1a311aa56365fe4dc3d664446f14d8c7835626e6a2a908be02eee3d403ad4808R44.